Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -351,7 +363,10 @@ open class ChatChannelVC: _ViewController,
         footerViewForMessage message: ChatMessage,
         at indexPath: IndexPath
     ) -> ChatMessageDecorationView? {
-        nil
+        guard message.id == lastReadMessageId, messages.first?.id != message.id, let channel = channelController.channel else { return nil }
+        let footer = components.unreadMessagesCounterDecorationView.init()
+        footer.content = channel
+        return footer
```

Comment:
I wonder if this is a breaking change 🤔 Since we are moving the view from a header to a footer 🤔 We already have some customers who are customizing the `unreadMessagesCounterDecorationView`, and are expecting to be rendered as a header, won't this be a problem?